test/extended/setup.sh

```diff
@@ -368,18 +366,13 @@ readonly CONFORMANCE_TESTS=(
 	"Pods should support retrieving logs from the container"
 	"Kubectl client Simple pod should support"
 	"Job should run a job to completion when tasks succeed"
-	"\[images\]\[mongodb\] openshift mongodb replication"
-	"\[job\] openshift can execute jobs controller"
-	"\[volumes\] Test local storage quota FSGroup"
-	"test deployment should run a deployment to completion"
 	"Variable Expansion"
 	"init containers"
 	"Clean up pods on node kubelet"
 	"\[Feature\:SecurityContext\]"
 	"should create a LimitRange with defaults"
 	"Generated release_1_2 clientset"
-	"\[Feature\:PodDisruptionbudget\]"
+	# "\[Feature\:PodDisruptionbudget\]" - does not match anything, perhaps intended to match tests in disruption.go ?
```

Comment:
@smarterclayton but i'm not sure what you'd like to do about this one.  I can change the focus to something that actually matches the tests in disruption.go, or i can delete it. your call.